src/components/reviews/reviewsList.jsx

```diff
@@ -7,30 +7,24 @@ class ReviewsList extends React.Component {
     super(props);
 
     this.state = {
-      reviewsShown: 0,
-      showMoreButton: true,
+      reviewsShown: 2,
       filters: new Set()
     };
-    this.loadMoreBound = this.loadMore.bind(this);
+    this.changeLoadBound = this.changeLoad.bind(this);
```

Comment:
I like that you change the name to [functionName]Bound after binding, i may take up this convention for this project